ajalibraries/ajantv2/src/ntv2driverinterface.cpp

```diff
@@ -990,7 +990,7 @@ bool CNTV2DriverInterface::BitstreamStatus (NTV2ULWordVector & outRegValues)
 void CNTV2DriverInterface::FinishOpen (void)
 {
 	// HACK! FinishOpen needs frame geometry to determine frame buffer size and number.
-	NTV2FrameGeometry fg;
+	NTV2FrameGeometry fg{};
```

Comment:
This initializer won't work if the library is built without C++11 compiler support.

Reply:
That's true, but I thought new versions of NTV2 were C++11+ only?

Reply:
Believe me, we'd love to pull that trigger, but we have a surprising number of customers that are stuck with older hardware and operating systems yet still need to use our newer hardware. In SDK 16.0, we changed to assume (by default) C++11 support, but the library can be built without it (by commenting out the lines defining the NTV2_USE_CPLUSPLUS11 and AJA_USE_CPLUSPLUS11 macros).

Reply:
In my own code I have preferred initializing NTV2 enum types to either their equivalent zero-value, or their UNKNOWN or INVALID value, ex:
```
NTV2FrameGeometry fg = NTV2_FG_1920x1080; // default 0-value for NTV2FrameGeometry
NTV2VideoFormat vf = NTV2_FORMAT_UNKNOWN; // default 0-value for NTV2VideoFormat
NTV2PixelFormat pf = NTV2_FBF_FIRST; // default 0-value for NTV2FrameBufferFormat/NTV2PixelFormat
```

Reply:
@mrbillaja Ahh, okay, I understand. That is unfortunate but understandable. In that case you can either do that or for other structs with less well-defined default values, it's often a safe bet to just do struct MyStruct Name = { 0 }; since 0 is also acceptable as an initializer for pointers and floats (though implicitly converted), and the compiler will zero out the rest as long as *something* is initialized explicitly.